Source/Runes.swift

```diff
@@ -50,3 +50,27 @@ infix operator -<< {
     // operator (`=`)
     precedence 100
 }
+
+/**
+compose two functions that produce results in a context, from left to right, returning a result in that context
+
+Expected function type: `(a -> m b) -> (b -> m c) -> a -> m c`
+*/
+infix operator >-> {
+    associativity right
+
+    // Same precedence as `>>-` and `-<<`.
+    precedence 100
+}
+
+/**
+like `>->`, but with the arguments flipped
```

Comment:
Maybe add the explicit documentation here as well, and move this note down. I'd be kinda annoyed to have to look up documentation for a different operator to see what this one does.